spec/integration/supplier_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe Vend::Resource::Supplier do
+  let(:expected_attributes) do
+    {
+      'id' => '6cb5c88c-3d5f-11e0-8697-4040f540b50a',
+      'name'   => 'Supplier Ben'
+    }
+  end
+
+  let(:expected_collection_length) { 1 }
+
+  it_behaves_like "a resource with a collection GET endpoint"
```

Comment:
This spec fails for me, maybe you forgot to add your `suppliers.json` file?
    1) Vend::Resource::Supplier behaves like a resource with a collection GET endpoint gets the collection
         Failure/Error: get_mock_response("#{file_path}.json")
         Errno::ENOENT:
           No such file or directory @ rb_sysopen - ../vend-ruby/spec/mock_responses/suppliers.json